the-blue-alliance-ios.xcodeproj/project.pbxproj

```diff
@@ -714,13 +726,14 @@
 				145C57BB1ED3C178004955B2 /* MatchBreakdownViewController.swift in Sources */,
 				92942DBB1E215B8D008E79CA /* MyTBAViewController.swift in Sources */,
 				924F8C871EC8C71C00F7447A /* NSManagedObjectContext+Extension.swift in Sources */,
-				924F8C8B1EC8D27B00F7447A /* RankingTableViewCell.swift in Sources */,
```

Comment:
Yeah, I almost made it one file, the reason I didn't is that it would make that one cell a little more complicated, and also that if we want to change one of them but not the other in the future, we won't have to worry about another sometimes-hidden cell feature.